kitodo-mediaserver-core/src/main/resources/schema.sql

```diff
@@ -33,3 +33,22 @@ CREATE TABLE `user` (
   PRIMARY KEY(`username`)
 );
 
+CREATE TABLE `action_data` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `work_id` varchar(255) NOT NULL,
+  `action_name` varchar(255) NOT NULL,
+  `request_time` TIMESTAMP NULL DEFAULT NULL,
+  `start_time` TIMESTAMP NULL DEFAULT NULL,
+  `end_time` TIMESTAMP NULL DEFAULT NULL,
+  PRIMARY KEY(`id`),
+  FOREIGN KEY (`work_id`) REFERENCES `work`(`id`)
+);
+
+CREATE TABLE `parameter` (
```

Comment:
Maybe we should call the table action_parameter

Reply:
The default would be "action_data_parameter" because it is part of ActionData. And if I change it in ActionData, the column names (key and value) will change too (action_paramater_key and action_paramater respectively action_data_parameter_key and action_data_parameter).